### DISCUSSION TITLE
Embedding Floating Point

### DISCUSSION THREAD

**Discussion by reza-putra** [milvus-io/milvus#16406] [Q&A and General discussion] [answered]
OS: Linux
SDK: Python

Problem: 

Hi there,

I use milvus with SKD pyton but I got problem with floating precision value from embedding value, I try to execute query with code below, 

```python
vector = col.query(
            expr = f"id in [{id}]",
            output_fields = ["id", "id_user", "embeddings"],
            consistency_level="Strong",
    )

print(vector[0]["embeddings"]
```

output: 

```
[0.063035, 0.388518, 0.744436, 0.40837, 0.713724] 
```

expect; 

```
[0.06303451955318451,0.3885182738304138,0.7444358468055725,0.40837016701698303,0.7137236595153809]
```

**Comment by yhmo** [comment, marked as answer]
In Milvus, embedding values are stored as float32 type, not float64.

**Reply by reza-putra** [reply]
We use Attu as Dashboard Milvus, they give us the embedding value with float64.

![Screenshot from 2022-04-07 11-49-36](https://user-images.githubusercontent.com/7883991/162122364-5cd18db8-a988-49ff-9e4f-50958f73ddf5.png)

**Reply by reza-putra** [reply]
And This value got from the same collection

**Reply by yhmo** [reply]
Milvus returns float32 values, the client convert float32 values to float64, I think the numbers are random after 7th place of decimal point.

**Reply by reza-putra** [reply]
owh i see, thank you for your help
